LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Molunerfinn & Spades
+Copyright (c) 2018-2019 Molunerfinn & Spades
```

Comment:
I think it should be `Copyright (c) 2018-2019 Molunerfinn & Spades & upupMing`

Reply:
It will be a great honor and encouragement, thanks and I'll add it!